stripe/webhook.py

```diff
@@ -17,6 +17,9 @@ class Webhook(object):
     def construct_event(
         payload, sig_header, secret, tolerance=DEFAULT_TOLERANCE, api_key=None
     ):
+        if api_key is None:
+            raise ValueError("api key is None")
+
```

Comment:
@ShreehariVaasishta The API key should not be needed for a webhook handler so I'm not entirely sure why this change is necessary. Can you clarify why you are raising an error here and what issue you are fixing with that change?